web/src/cypress/support/index.ts

```diff
@@ -36,9 +34,9 @@ import './sql'
 import './form'
 import './dialog'
 
-Cypress.env('RETRIES', 2)
+Cypress.env('retries', 1)
```

Comment:
Was it intentional to change retries to 1, down from 2?

Reply:
yep! thats what the migration doc says. https://docs.cypress.io/guides/references/migration-guide.html#Configure-test-retries-in-the-configuration-file

Reply:
It looks like their example is just misleading, we want 3 attempts and we'd still need to set retries to 2 in order to get that behavior -- however it looks like we can delete this line and instead add `"retries": 2` to `cypress.json` based on that doc you linked.

Reply:
gotcha, ok, all fixed!